coresimd/src/x86/i586/avx.rs

```diff
@@ -607,77 +607,77 @@ pub unsafe fn _mm256_xor_ps(a: f32x8, b: f32x8) -> f32x8 {
 }
 
 /// Equal (ordered, non-signaling)
-pub const _CMP_EQ_OQ: u8 = 0x00;
```

Comment:
The unsigned part was on purpose (bit flags consistently used unsigned integers), why we use `8` instead of `32` bits here, I don't know. 

Whatever we do, we should do it consistently, and have a guideline for it.

Reply:
So IIRC as mentioned below, arguments that can only be `8`-bit long, `16` bit long, etc. were made `_8` (e.g. `i8`) on purpose. This happens on most intrinsics taking an `imm8` or `imm16`.

The Intel intrinsics use here `int`/`long`/... very loosely, so this was part of offering better types by default.